### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by Anish2**

### CODE REVIEW

**Comment by illegalprime** [comment]
@Anish2 you must have these options configurable in some way: https://github.com/HackGT/SponsorshipPortal/blob/990e5a9848977a10d59d47ad453a1ec0e48adbd8/backend/app/controllers/LoginController.go#L23

File: frontend/src/js/index.jsx [hunk 1]
```diff
@@ -24,3 +25,7 @@ export default store;
 // Particle Js Decorations
 window.particlesJS('particle', particlesOption);
```
Comment (illegalprime) [inline frontend/src/js/index.jsx]: We should probably get rid of this, it's not so professional. Thoughts @bunsenmcdubbs ?

Comment (RSLi) [reply frontend/src/js/index.jsx]: It's for decoration purpose. The configuration can be customized here: http://vincentgarreau.com/particles.js/. If it looks bad then I should probably remove it.

Comment (bunsenmcdubbs) [reply frontend/src/js/index.jsx]: I agree with Michael - we should remove it. Seems irrelevant

**Comment by illegalprime** [comment]
@RSLi separate your new UI changes in another branch so we can merge this PR.

**Comment by RSLi** [comment]
@illegalprime The actual new UI changes are in another branch already. The entire frontend directory in this PR will be abandoned. I'll be pushing the new frontend tomorrow.

**Comment by illegalprime** [comment]
@RSLi thanks!

**Comment by Anish2** [comment]
Can this be merged now?

**Comment by illegalprime** [comment]
Closing because this was manually merged by @bunsenmcdubbs .

**Pull request closed without merging**